invenio_users_resources/errors.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 KTH Royal Institute of Technology
+# Copyright (C) 2022 Northwestern University.
+#
+# Invenio-Communities is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Users resources base error."""
+
+from flask_babelex import gettext as _
+
+
+class UsersError(Exception):
```

Comment:
I think we don't need these base exceptions for now...Let's keep it simple and if we need to have base classes we can add them later, what do you think? If it's fine with you, I think you can remove this file...

Reply:
I add the base class to keep it consistent with other packages like invenio-communities, but I agree that it is not needed now.